api/v1/apimodel.go

```diff
@@ -214,6 +214,8 @@ type ServiceDto struct {
 	CommitHash string `json:"commitHash" yaml:"-"`
 	// The jira issue to use for committing a change, or the last jira issue used.
 	JiraIssue string `json:"jiraIssue" yaml:"-"`
+	// A short description of the functionality of the service.
+	Description *string `json:"description,omitempty" yaml:"description,omitempty"`
```

Comment:
what about ServiceCreateDto and ServicePatchDto?

Must also adapt the mappings for these cases.